src/Options/Definitions.js

```diff
@@ -3,626 +3,611 @@
-  },
-  push: {
-    env: 'PARSE_SERVER_PUSH',
-    help:
-      'Configuration for push, as stringified JSON. See http://docs.parseplatform.org/parse-server/guide/#push-notifications',
-    action: parsers.objectParser,
-  },
-  readOnlyMasterKey: {
-    env: 'PARSE_SERVER_READ_ONLY_MASTER_KEY',
-    help: 'Read-only key, which has the same capabilities as MasterKey without writes',
-  },
-  restAPIKey: {
-    env: 'PARSE_SERVER_REST_API_KEY',
-    help: 'Key for REST calls',
-  },
-  revokeSessionOnPasswordReset: {
-    env: 'PARSE_SERVER_REVOKE_SESSION_ON_PASSWORD_RESET',
-    help:
-      "When a user changes their password, either through the reset password email or while logged in, all sessions are revoked if this is true. Set to false if you don't want to revoke sessions.",
-    action: parsers.booleanParser,
-    default: true,
-  },
-  scheduledPush: {
-    env: 'PARSE_SERVER_SCHEDULED_PUSH',
-    help: 'Configuration for push scheduling, defaults to false.',
-    action: parsers.booleanParser,
-    default: false,
-  },
-  schemaCacheTTL: {
-    env: 'PARSE_SERVER_SCHEMA_CACHE_TTL',
-    help:
-      'The TTL for caching the schema for optimizing read/write operations. You should put a long TTL when your DB is in production. default to 5000; set 0 to disable.',
-    action: parsers.numberParser('schemaCacheTTL'),
-    default: 5000,
-  },
-  serverCloseComplete: {
-    env: 'PARSE_SERVER_SERVER_CLOSE_COMPLETE',
-    help: 'Callback when server has closed',
-  },
-  serverStartComplete: {
-    env: 'PARSE_SERVER_SERVER_START_COMPLETE',
-    help: 'Callback when server has started',
-  },
-  serverURL: {
-    env: 'PARSE_SERVER_URL',
-    help: 'URL to your parse server with http:// or https://.',
-    required: true,
-  },
-  sessionLength: {
-    env: 'PARSE_SERVER_SESSION_LENGTH',
-    help: 'Session duration, in seconds, defaults to 1 year',
-    action: parsers.numberParser('sessionLength'),
-    default: 31536000,
-  },
-  silent: {
-    env: 'SILENT',
-    help: 'Disables console output',
-    action: parsers.booleanParser,
-  },
-  startLiveQueryServer: {
-    env: 'PARSE_SERVER_START_LIVE_QUERY_SERVER',
-    help: 'Starts the liveQuery server',
-    action: parsers.booleanParser,
-  },
-  userSensitiveFields: {
-    env: 'PARSE_SERVER_USER_SENSITIVE_FIELDS',
-    help:
-      'Personally identifiable information fields in the user table the should be removed for non-authorized users. Deprecated @see protectedFields',
-    action: parsers.arrayParser,
-  },
-  verbose: {
-    env: 'VERBOSE',
-    help: 'Set the logging to verbose',
-    action: parsers.booleanParser,
-  },
-  verifyUserEmails: {
-    env: 'PARSE_SERVER_VERIFY_USER_EMAILS',
-    help: 'Enable (or disable) user email validation, defaults to false',
-    action: parsers.booleanParser,
-    default: false,
-  },
-  webhookKey: {
-    env: 'PARSE_SERVER_WEBHOOK_KEY',
-    help: 'Key sent with outgoing webhook calls',
-  },
+  "accountLockout": {
```

Comment:
It is a minor item, but why do we have all this changes?

Reply:
Good question, this has to be a pre-commit hook or something on my side. I think it is prettier or lint.

Reply:
Oh, these changes actually seem to occur for me on `npm run definitions`. The file is then built with double quotes. Could you try this out on your side, if that happens as well?

Reply:
It seems that the generator already adds the double quotes before writing to file, as you can see when inspecting the `res` output:

https://github.com/parse-community/parse-server/blob/25fb576776fefee631b457e358a2f567867c9c80/resources/buildConfigDefinitions.js#L279

This [babel issue](https://github.com/babel/babel/issues/4109) describes that the quotes option has also been removed and babel outputs with double quotes. So much about babel's side at least.

Reply:
OK, when I run `npm run prettier` the double quotes are removed. That also affects some other files though, which are not related to this PR. What is the current workflow - should we have prettier run as a pre-commit hook? It seems that currently a PR is not guaranteed to be merged with prettier, if the author did not manually run it?